plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/hms/TestTrinoHive4CatalogWithHiveMetastore.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.hms;
+
+import io.trino.plugin.hive.containers.Hive4MinioDataLake;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
+
+public class TestTrinoHive4CatalogWithHiveMetastore
+        extends TestTrinoHiveCatalogWithHiveMetastore
+{
+    @Override
+    HiveMinioDataLake hiveMinioDataLake()
```

Comment:
We could add a constructor instead.

Reply:
This wasn't possible as it requires two constructors(with HiveMinioDataLake parameter and a default one). However, test framework doesn't seem to support multiple constructors.